index.html

```diff
@@ -49,9 +74,38 @@
           
         </div> 
       </nav>
+
+    
+
+<div class="deskdiv"></div>
+      <!-- <div class="pop-up-desk" id="popupdesk">
+        <div class="pop-up-header-desk">
+          <h2>Keeping track of hundreds  of components website</h2>
+          <span class="close-desk" id="close-desk-">&times;</span>
+        </div>
+        <div>
+          <ul class="pop-up-tags-desk">
+            <li>html</li>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+          </ul>
+        </div>
+        <div class="desk-popup-img">
+        <img src="Images\desktop-popup.png" alt="popup-img.png">
+        </div>
+        <div class="popup-description-desk">
+          <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.</p>
+        </div>
+        <div class="popup-buttons-desk">
+          <button id="see-live-desk" class="see-live">See Live <img src="Images\mob-popup.png" alt="img.png"></button>
+          <button id="see-source-desk" class="see-source">See Source <img src="Images\pop-icon-github.png" alt="img.png"></button>
+        </div>
+    </div> -->
```

Comment:
It's highly recommended to keep your repository as clean as possible, and removing the unused commented code will help you keep it clean. Kindly make sure to remove it.